src/client/client.ts

```diff
@@ -87,7 +87,7 @@ export function activate(context: ExtensionContext): void {
     window.registerTreeDataProvider('q-explorer', qRoot);
     window.registerTreeDataProvider('q-history', qHistory);
     qHistory.refresh();
-    QueryConsole.createOrShow();
+    QueryConsole.get();
```

Comment:
Please add a show after this line. I want vscode to show q output when vscode-q is activated.